File: server/php/src/Infrastructure/Mappers/PaymentDetails/PaymentProduct3208SpecificOutputMapper.php
<?php

namespace OnlinePayments\ExampleApp\Infrastructure\Mappers\PaymentDetails;

use OnlinePayments\ExampleApp\Application\Domain\Payments\PaymentDetails\PaymentProduct3208SpecificOutput as PaymentProduct3208SpecificOutputDto;
use OnlinePayments\Sdk\Domain\PaymentProduct3208SpecificOutput as PaymentProduct3208SpecificOutputSdk;

final class PaymentProduct3208SpecificOutputMapper
{
    public static function mapFromSdkResponse(?PaymentProduct3208SpecificOutputSdk $response): ?PaymentProduct3208SpecificOutputDto
    {
        if ($response === null) {
            return null;
        }

        $dto = new PaymentProduct3208SpecificOutputDto();
        $dto->buyerCompliantBankMessage = $response?->buyerCompliantBankMessage ?? null;

        return $dto;
    }
}

File: server/dotnet/sdk-dotnet-example/Infrastructure/Mappers/PaymentDetails/PaymentProduct3208SpecificOutputMapper.cs
using PaymentProduct3208SpecificOutputSdk = OnlinePayments.Sdk.Domain.PaymentProduct3208SpecificOutput;
using PaymentProduct3208SpecificOutputDto = Business.Domain.Payments.PaymentDetails.PaymentProduct3208SpecificOutput;

namespace Infrastructure.Mappers.PaymentDetails;

public static class PaymentProduct3208SpecificOutputMapper
{
    public static PaymentProduct3208SpecificOutputDto Map(PaymentProduct3208SpecificOutputSdk? response)
    {
        return new()
        {
            BuyerCompliantBankMessage = response?.BuyerCompliantBankMessage
        };
    }
}